dev-lua/lua-utf8/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
you can't add an unmaintained package.
you have to specify yourself as first maintainer, and a project or a person with `@gentoo.org` address as second maintainer.
usually it's `proxy-maint`, but they are not taking new packages lately, you can add me.

Reply:
fixed